app/controllers/people_controller.rb

```diff
@@ -3,6 +3,8 @@
 #   the COPYRIGHT file.
 
 class PeopleController < ApplicationController
+  include PhotosHelper
```

Comment:
Helpers are meant for the view. For controllers move logic to a model (class) method or a service object.